src/Traits/AdditionalAssertions.php

```diff
@@ -66,10 +66,12 @@ public function assertActionUsesMiddleware($controller, $method, $middleware = n
             PHPUnitAssert::assertNotNull($route, 'Unable to find route for controller action (' . $controller . '@' . $method . ')');
         }
 
+        $usedMiddlewares = array_diff($route->gatherMiddleware(), $route->action['excluded_middleware']);
```

Comment:
Is the `excluded_middleware` key always defined?

Reply:
I'm not sure, added a null coalescing operator in case it's not.